apps/arm/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ target_link_libraries(arm.${TARGET}-board.elf
         LookupTable
         Motor
         uwrt-mars-rover-hw-bridge
+	PwmIn
```

Comment:
fix spacing

Reply:
I'm not seeing this indentation on both vim and vscode. I think it may just be a github thing.